### DISCUSSION TITLE
How to pull data

### DISCUSSION THREAD

**Discussion by christianaz** [FrandsenGroup/beams#76] [Q&A] [answered]
Hello! Quick question. GitHub keeps sending me messages that using passwords to pull won't work soon. What is the best way to pull the code without using the GitHub username and password? The article I looked at was talking about some sort of token, but I'm not quite sure what that is.

**Comment by benfrandsen** [comment, marked as answer]
I got this message too. I just followed the directions given in the email (or at the link provided in the email) and was able to set it up. It's kind of a pain, but it wasn't too bad. I think you need to follow the directions for "developer".

**Comment by aPeter1** [comment]
I didn't get this message so I'm not sure what's up with that, were you able to figure it out?

**Reply by christianaz** [reply]
I think so? I made the token thing it asked for, but I'm not quite sure how it works. I thought it was just an alternate form of the password, but it wasn't quite working. I'll keep looking at it.
